fr-FR.lproj/Localizable.strings

```diff
@@ -0,0 +1,369 @@
+"Installed OpenSSH.\n" = "OpenSSH installé.\n"
+"Removing Cydia Dummy Package..." = "Suppression du package factice Cydia..."
+"Unable to remove Cydia Dummy Package." = "Impossible de supprimer le package factice Cydia."
+"Removed Cydia Dummy Package.\n" = "Paquet factice Cydia supprimé.\n"
+"Removing Electra's Cydia Upgrade Helper..." = "Retrait de l'auxiliaire de mise à niveau Cydia d'Electra..."
+"Unable to remove Electra's Cydia Upgrade Helper." = "Impossible de supprimer Cyd Upgrade Helper d'Electra."
+"Installing Cydia..." = "Installation de Cydia..."
+"Unable to get Cydia version." = "Impossible d'obtenir la version de Cydia."
+"Unable to reinstall Cydia." = "Impossible de réinstaller Cydia."
+"Installed Cydia.\n" = "Cydia installé.\n"
+"Loading Daemons..." = "Loading des émulateurs..."
+"Loaded Daemons.\n" = "Émulateurs chargés.\n"
+"Resetting Cydia cache..." = "Réinitialisation du cache Cydia..."
+"Unable to clean Cydia's directory." = "Impossible de nettoyer le répertoire de Cydia."
+"Unable to clean Cydia's cache directory." = "Impossible de nettoyer le répertoire de cache de Cydia."
+"Reset Cydia Cache.\n" = "Réinitialiser le cache Cydia.\n"
+"Refreshing icon cache..." = "Actualisation du cache d'icônes..."
+"Unable to refresh icon cache." = "Impossible d'actualiser le cache des icônes."
+"Ran uicache.\n" = "Ran uicache.\n"
+"Flushing preference cache..." = "Cache de préférences de vidage..."
+"Unable to flush preference cache." = "Impossible de vider le cache de préférences."
+"Flushed preference cache.\n" = "Cache de préférences vidé.\n"
+"Loading Tweaks..." = "Chargement des Tweaks..."
+"Unable to load tweaks." = "Impossible de charger les tweaks."
+"Loaded Tweaks.\n" = "Tweaks chargés.\n"
+"Deinitializing jailbreak..." = "Déinitialisation du jailbreak..."
+"Unable to make my task a non-platform task." = "Impossible de transformer ma tâche en tâche non-plate-forme."
+"Unable to make my codesign blob a non-platform blob." = "Impossible de transformer mon objet blob en codesign en un objet non-plateforme."
+"Unable to drop kernel's credentials." = "Impossible de supprimer les informations d'identification du noyau."
+"Unable to set user id." = "Impossible de supprimer les informations d'identification du noyau."
+"Unable to verify user id." = "Impossible de vérifier l'identifiant de l'utilisateur."
+"Unable to restore shenanigans in kernel memory." = "Impossible de restaurer les manigances dans la mémoire du noyau."
+"Unable to deallocate new host port." = "Impossible de libérer le nouveau port hôte."
+"Unable to deallocate my original host port." = "Impossible de désallouer mon port hôte d'origine."
+"Jailbroken" = "Jailbroken"
+"Unable to create success file." = "Impossible de créer le fichier de réussite."
+"Jailbreak Completed with Status:" = "Jailbreak terminé avec statut:"
+"The device will now respring." = "L'appareil va maintenant respring."
+"The app will now exit." = "L'application va maintenant quitter."
```

Comment:
L'application va maintenant se fermer.

Reply:
"Unable to create success file." = "Impossible de créer le fichier de réussite."
"Jailbreak Completed with Status:" = "Jailbreak fini configuration suivante:"
"The device will now respring." = "L'appareil va maintenant respring."
"The app will now exit." = "L'app va maintenant se fermer ."

je suis pas sur du deuxième mais bon